### PR TITLE
Make PO feature description more business-focused

### PR DESCRIPTION
**Pull request by CLEckhardt**
Changes the PO feature overview to be less technical and more
business-focused. This brings it in line with the other feature
overviews.

Signed-off-by: Chris Eckhardt <eckhardt@bitwise.io>

### CODE REVIEW

**Review by dnewh — DISMISSED** [review]

File: docs/0.3/grid_purchase_order.md [hunk 1]
```diff
@@ -1,158 +1,62 @@
...
+
+As mentioned above, organizations may adopt Grid Purchase Order as the system
+of record between them and their partner organizations. In this case, their
+technology integration teams will likely enable users to interact with Grid
+through some integration application.
+
+Organizations that choose to use their existing systems of record can use Grid
+Purchase Order to synchronize the purchase order records of them and their
+partners. In this case, their technology teams will integrate Grid directly
+with their systems of record.
```
Comment (shannynalayna) [inline docs/0.3/grid_purchase_order.md:62]: I think at some point in this doc, we should also mention the XML validation similar to how we mention this for the product feature. Definitely something we should highlight here, moreso the part of using GS1 standards for the order XML

Comment (shannynalayna) [reply docs/0.3/grid_purchase_order.md:62]: That could probably go in the "how this works" section

Comment (CLEckhardt) [reply docs/0.3/grid_purchase_order.md:66]: Added

File: docs/0.3/grid_purchase_order.md [hunk 2]
```diff
@@ -1,158 +1,66 @@
 # Purchase Order Overview
```
Comment (shannynalayna) [inline docs/0.3/grid_purchase_order.md:2]: There should be a copyright on this doc:
```
<!--
  Copyright 2022 Cargill Incorporated
  Licensed under Creative Commons Attribution 4.0 International License
  https://creativecommons.org/licenses/by/4.0/
-->
```

**Review by shannynalayna — APPROVED** [review]

**Review by leebradley — APPROVED** [review]

**Comment by CLEckhardt** [comment]
Rendered: https://github.com/CLEckhardt/grid-docs/blob/update_po_feature/docs/0.3/grid_purchase_order.md

**Review by leebradley — APPROVED** [review]

File: docs/0.3/grid_purchase_order.md [hunk 3]
```diff
@@ -1,158 +1,194 @@
...
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Grid Purchase Order is a smart contract that enables buyers and sellers to
+automate and synchronize their purchase order processes and data. It aims
+to reduce administrative overhead, improve transaction accuracy and speed, and
+reduce the number of disputes, ultimately improving business relationships.
+
+### Contents
```
Comment (vaporos) [inline docs/0.3/grid_purchase_order.md:14]: This section should be removed, it duplicates the right sidebar.

File: docs/0.3/grid_purchase_order.md [hunk 4]
```diff
@@ -1,158 +1,194 @@
...
+Purchase Order contract represents a negotiated purchase order between a buyer
+and seller throughout its lifecycle and contains all its data, including its
+history. Purchase Order uses the Grid Workflow feature to mirror real-world
+business processes, defining the possible business states through which
+contracts can flow, the criteria that must be met to move to the next state,
+and the actions each party can take in each state. It also leverages Grid Pike
+to configure role-based access control.
+
+### Key benefits
+- A complete solution for the purchase order process
```
Comment (vaporos) [inline docs/0.3/grid_purchase_order.md:56]: Add blank line after the header.

File: docs/0.3/grid_purchase_order.md [hunk 5]
```diff
@@ -1,158 +1,194 @@
...
+more involved business scenarios, such as where product is out of stock or
+requested delivery dates must be adjusted, trade partners interact to reach
+agreement and modify purchase order records.
+
+Today, the communication of order information occurs in various ways: by phone,
+email, SMS, eCommerce marketplaces, Electronic Data Interchange (EDI), etc.
+Both the manual coordination and automated, point-to-point sharing of
+purchasing information between trade partners present challenges within
+day-to-day supply chain operations including but not limited to:
+- poor data accuracy stemming from manual data entry errors,
```
Comment (vaporos) [inline docs/0.3/grid_purchase_order.md:88]: Add blank line before the list.

File: docs/0.3/grid_purchase_order.md [hunk 6]
```diff
@@ -1,158 +1,194 @@
...
+Purchase Order addresses the challenges outlined above by enabling trade
+partners to directly  collaborate on the creation and modification of a single
+purchase order contract, with all partners enjoying a real-time view of the
+state of the order.
+
+### Expected outcomes of using Purchase Order
+
+The primary outcomes we expect organizations using Grid Purchase Order to see
+are: 
+- **Improved cost efficiency**: An organization’s financial results can benefit
```
Comment (vaporos) [inline docs/0.3/grid_purchase_order.md:114]: Add a blank line before the list.

File: docs/0.3/grid_purchase_order.md [hunk 7]
```diff
@@ -1,158 +1,194 @@
...
+
+Purchase Order offers solutions for both simple and complex collaboration needs
+by defining two built-in Purchase Order designs. These designs leverage Grid
+Workflow, which requires the use of a state transition model, to define the
+states through which a purchase order may flow. The first was inspired by
+vendor-managed purchasing relationships, and the second by a more traditional
+procurement relationship between a buyer and a seller.
+
+Purchase Order workflows use three core components to facilitate transactions:
+- workflow states - these represent the steps of the business process
```
Comment (vaporos) [inline docs/0.3/grid_purchase_order.md:176]: Add blank line before list.

**Review by vaporos — CHANGES_REQUESTED** [review]

File: docs/0.3/grid_purchase_order.md (hunk 7 above)
Comment (CLEckhardt) [reply docs/0.3/grid_purchase_order.md:168]: Fixed

File: docs/0.3/grid_purchase_order.md (hunk 6 above)
Comment (CLEckhardt) [reply docs/0.3/grid_purchase_order.md:105]: Fixed

File: docs/0.3/grid_purchase_order.md (hunk 5 above)
Comment (CLEckhardt) [reply docs/0.3/grid_purchase_order.md:78]: Fixed

File: docs/0.3/grid_purchase_order.md (hunk 4 above)
Comment (CLEckhardt) [reply docs/0.3/grid_purchase_order.md:45]: Fixed

File: docs/0.3/grid_purchase_order.md (hunk 3 above)
Comment (CLEckhardt) [reply docs/0.3/grid_purchase_order.md]: Thanks, fixed

**Review by vaporos — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**